tests-legacy/Endpoints/AjaxProductListTest.php

```diff
@@ -37,6 +37,7 @@ protected function setUp()
     {
         parent::setUp();
         $this->employeeLogin();
+        $_SERVER['REQUEST_URI'] = _PS_ROOT_DIR_ . '/admin-dev/ajax_products_list.php';
```

Comment:
Why do you need to change this value if the tests were working previously?

Reply:
they were no longer working after changing the security system